latexify/core.py

```diff
@@ -3,9 +3,31 @@
 import ast
 import math
 import inspect
+from typing import Callable
 
 
 class LatexifyVisitor(ast.NodeVisitor):
+
+  def __init__(self, math_symbol=True):
+    self.math_symbol = math_symbol
+    super(ast.NodeVisitor).__init__()
+
+  def _parse_math_symbols(self, val: str) -> str:
+    if not self.math_symbol:
+      return val
+    greek_and_hebrew = [
+        'aleph', 'alpha', 'beta', 'beth', 'chi', 'daleth',
+        'delta', 'digamma', 'epsilon', 'eta', 'gamma', 'gimel',
+        'iota', 'kappa', 'lambda', 'mu', 'nu', 'omega', 'omega',
+        'phi', 'pi', 'psi', 'rho', 'sigma', 'tau', 'theta',
+        'upsilon', 'varepsilon', 'varkappa', 'varphi', 'varpi', 'varrho',
+        'varsigma', 'vartheta', 'xi', 'zeta'
+    ]
+    if val.lower() in greek_and_hebrew:
```

Comment:
`lower()` looks a too generalized behavior. I think we need to process only exact matches.